js/piwik.js

```diff
@@ -1138,15 +1138,7 @@ if (typeof Piwik !== 'object') {
             }
 
             function _elementInDocument(element) {
-                element = element.parentNode;
-
-                while (element) {
-                    if (element === documentAlias) {
-                        return true;
-                    }
-                    element = element.parentNode;
-                }
-                return false;
+                return documentAlias.documentElement.contains(element);
```

Comment:
IE5 sounds good, even IE6 support and greater is/  would be enough for JavaScript tracker